File: samples/server/petstore/jaxrs-cxf-cdi/src/main/java/io/swagger/api/impl/StoreApiServiceImpl.java
package io.swagger.api.impl;

import io.swagger.api.*;
import io.swagger.model.*;

import org.apache.cxf.jaxrs.ext.multipart.Attachment;

import java.util.Map;
import io.swagger.model.Order;

import java.util.List;

import java.io.InputStream;

import javax.enterprise.context.RequestScoped;
import javax.ws.rs.core.Response;
import javax.ws.rs.core.SecurityContext;

@RequestScoped
@javax.annotation.Generated(value = "class io.swagger.codegen.languages.JavaJAXRSCXFCDIServerCodegen", date = "2016-10-11T07:40:42.070+01:00")
public class StoreApiServiceImpl implements StoreApiService {
      @Override
      public Response deleteOrder(String orderId, SecurityContext securityContext) {
      // do some magic!
      return Response.ok().entity("magic!").build();
  }
      @Override
      public Response getInventory(SecurityContext securityContext) {
      // do some magic!
      return Response.ok().entity("magic!").build();
  }
      @Override
      public Response getOrderById(Long orderId, SecurityContext securityContext) {
      // do some magic!
      return Response.ok().entity("magic!").build();
  }
      @Override
      public Response placeOrder(Order body, SecurityContext securityContext) {
      // do some magic!
      return Response.ok().entity("magic!").build();
  }
}

File: samples/server/petstore/kotlin-server/javalin-6/src/main/kotlin/org/openapitools/server/apis/StoreApiServiceImpl.kt
package org.openapitools.server.apis

import org.openapitools.server.models.Order
import io.javalin.http.Context

class StoreApiServiceImpl : StoreApiService {

    override fun deleteOrder(orderId: String, ctx: Context): Unit {
        TODO("Implement me")
    }

    override fun getInventory(ctx: Context): Map<String, Int> {
        TODO("Implement me")
    }

    override fun getOrderById(orderId: Long, ctx: Context): Order {
        TODO("Implement me")
    }

    override fun placeOrder(order: Order, ctx: Context): Order {
        TODO("Implement me")
    }
}
